Week1/Day1/Exercises XP Ninja/code.py

```diff
@@ -0,0 +1,42 @@
+#You can call python3 from any directory because it's typically installed as a
+# system-wide executable. This means that the operating system knows where to find the
+# python3 executable regardless of your current directory.
+
+
```

Comment:
what about PATH and alias?

Reply:
The PATH environment variable is essential for running commands and scripts from any location in the Command Prompt or PowerShell without specifying the full path to the executable file
In CMD or shell environments, aliases are shortcuts or alternate names for commands. Users can define custom aliases to abbreviate commonly used commands or to create shortcuts for longer commands.